protocol/messenger.go

```diff
@@ -1975,6 +1975,7 @@ func (m *Messenger) AddAdminsToGroupChat(ctx context.Context, chatID string, mem
 	return m.addMessagesAndChat(chat, buildSystemMessages([]v1protocol.MembershipUpdateEvent{event}, m.systemMessagesTranslations), &response)
 }
 
+// Kept only for backward compatibility, explicit join has been removed
```

Comment:
I think we can drop it, we only need to be backward compatible protocol between users, but this will ship with the application (so it's a "local" breaking change, which is ok, while we want to have backward compatibility with peers)

Reply:
oh yes you are rightt, sorry missed that we use this internally, thanks!